core/src/display_object/stage.rs

```diff
@@ -386,8 +390,20 @@ impl<'gc> Stage<'gc> {
     /// Set the stage alignment.
     /// This only has an effect if the scale mode is not `StageScaleMode::ExactFit`.
     pub fn set_align(self, context: &mut UpdateContext<'_, 'gc>, align: StageAlign) {
-        self.0.write(context.gc_context).align = align;
-        self.build_matrices(context);
+        if !self.forced_align() {
```

Comment:
just to be sure: this can be observed from AS2/3?
As in, if it's in forced align mode:
```as
stage.align = "B";
trace(stage.align); // does this trace "B" or the preset forced alignment?
```

Reply:
Yes it prevents setting the property so it will remain whatever it was locked to. And forceScale behaves the same way.